src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -36,6 +36,14 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
         }];
     }
 
+
+    // Taken from emitter.ts L4798
+    function formatSynthesizedComment(comment: SynthesizedComment) {

```

Comment:
I copied this function directly from `emitter.ts` because I needed this exact feature. I wish it were exposed as a utility. I feel like there has to be a better way than rewriting the implementation.